pontus_perception/launch/yolo.launch.py

```diff
@@ -0,0 +1,48 @@
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+import os
+import sys
+import yaml
+
+def generate_launch_description():
+
+    pkg_share = get_package_share_directory('pontus_perception')
+
+    auv_arg = DeclareLaunchArgument('auv')
+    auv_config = LaunchConfiguration('auv')
+
+    auv_config_str = None
+    for arg in sys.argv:
+        if arg.startswith('auv:='):
+            auv_config_str = arg.split(':=')[1]
+    
+    camera_data = None
+    with open(f'{pkg_share}/config/{auv_config_str}/camera_config.yaml', 'r') as stream:
```

Comment:
Can you create a `camera_config.yaml` file with the same structure as that in `virtuoso_perception`. For now, there should just be one OAK-D camera in the config.